milabowl-astro/src/layouts/Layout.astro

```diff
@@ -1,28 +1,32 @@
 ---
-import '@fontsource/inter/400.css';
+import "@fontsource/inter/400.css";
 import "@fontsource/inter/700.css";
-import Navbar from './Navbar.astro';
+import Navbar from "./Navbar.astro";
 
 export interface Props {
-	title: string;
+  title: string;
 }
 
 const { title } = Astro.props;
 ---
 
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/svg+xml" href="/favicon.ico" />
-		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
-	</head>
-	<body class="bg-slate-100 dark:bg-slate-900">
-		<Navbar/>
-		<div class="max-w-screen-2xl mx-auto">
-			<slot/>
-		</div>
-	</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+    <meta name="generator" content={Astro.generator} />
+    <title>{title}</title>
+  </head>
+  <body class="bg-slate-100 dark:bg-slate-900">
+    <Navbar />
+    <div class="max-w-screen-2xl mx-auto">
+      <slot />
+    </div>
+    <script
+      is:inline
+      src="https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.8.0/flowbite.min.js"
```

Comment:
Ikke helt fan av å hente inn via CDN på denne måten, men kanskje den enkleste måten for denne bruken (bare behold den slik).